lib/worldwide/address.rb

```diff
@@ -147,7 +147,14 @@ def valid?
 
     RESERVED_DELIMITER = "\u00A0"
     def concatenate_address1
-      additional_fields = region.combined_address_format["address1"] || []
+      detected_scripts = Worldwide::Scripts.identify(text: street_name).map(&:to_s)
```

Comment:
Right now in ruby we're only doing this check on `street_name`, but in typescript we're doing it on any field defined in `address1`.